03-types-of-data.Rmd

```diff
@@ -273,7 +273,7 @@ Methods Discussed:
 <iframe width=100% height=800px src='images/03-PoliceViolenceIncidents_BC.html' frameBorder="0" >
   <p>Your browser does not support iframes</p>
 </iframe>
-
+Data from City of Vancouver and licensed under the Open Government License - Vancouver
```

Comment:
This statement should be omitted

Reply:
June indicated that this police violence web map will be removed from the chapter, so this is moot.